client/src/components/Navbar/Navbar.scss

```diff
@@ -126,7 +125,8 @@ a {
   margin-left: auto;
   margin-right: 15px;
 
-  filter: invert(48%) sepia(16%) saturate(1926%) hue-rotate(230deg) brightness(86%) contrast(84%);
+  // filter: invert(48%) sepia(16%) saturate(1926%) hue-rotate(230deg) brightness(86%) contrast(84%);
+  filter: var(--purple-icon-filter);
```

Comment:
Let's actually move away from filters, it's really bad style
Could you color the svg files themselves?

Reply:
and if possible, color all svg currently in the repo
You can edit the svg path code or use third party tool such as Adobe illustrator